clients/src/main/java/org/apache/kafka/common/KafkaFuture.java

```diff
@@ -127,8 +127,6 @@ public static KafkaFuture<Void> allOf(KafkaFuture<?>... futures) {
     public abstract <R> KafkaFuture<R> thenApply(BaseFunction<T, R> function);
 
     /**
-     * @see KafkaFuture#thenApply(BaseFunction)
```

Comment:
Why did we link to itself? Interesting

Reply:
It links to the method above but we also link to the other method on the line just below.